vendor/engines/secure_rooms/config/routes.rb

```diff
@@ -4,5 +4,7 @@
       facility_product_routing_concern
       resources :price_policies, controller: "secure_room_price_policies", except: :show
     end
+
+    resources :indala_numbers, controller: "secure_rooms/indala_numbers", only: [:edit, :update]
```

Comment:
I could be persuaded, especially if it's too complicated, but could we make the path for this `/facilities/:facility_id/users/:id/indala_numbers`?